config-service/src/main/resources/configs/common/application.conf

```diff
@@ -2,6 +2,8 @@ main.class = org.hypertrace.config.service.ConfigService
 service.name = config-service
 service.port = 50101
 service.admin.port = 50102
+global.config.service.port.internal = 50103
```

Comment:
I can remove the internal suffix if you prefer - but this was meant to convey it is an internal port

Reply:
all of the other ones above it are internal too though - removing it would be my preference.

Reply:
done